src/Options.ts

```diff
@@ -27,6 +27,16 @@ const options = {
     description: 'Filter by a component name regex pattern',
     type: 'string' as const,
   },
+  'output-sarif-files': {
+    default: false,
+    description: 'Should Sarif files be outputted',
+    type: 'boolean' as const,
+  },
+  'sarif-output-dir': {
+    default: 'sarif-test-results',
+    description: 'Directory to output Sarif files to',
```

Comment:
Would it make sense to output the results to stdout, and then users can redirect that to a file if they want?

I'm imagining eventually having the concept of formats, and being able to do something like

```sh
axe-storybook --format sarif > sarif-output.sarif
axe-storybook --format mocha > mocha-output.txt
# Etc.
```

Although there's a bunch of work to do to make that happen 😬

Reply:
Maybe we go with your `sarif-output-dir`, and in v2 I'll plan on introducing different formats instead (as long as that will still meet your needs).